microscopium/preprocess.py

```diff
@@ -569,6 +578,94 @@ def find_background_illumination(fns, radius=51, quantile=0.05,
     return illum
 
 
+def correct_multiimage_illumination(im_fns, illum, stretch_quantile=0,
+                                    random_state=None):
+    """Divide input images pointwise by the illumination field.
+
+    However, where `correct_image_illumination` rescales each individual
+    image to span the full dynamic range of the data type, this one
```

Comment:
Maybe a reference to this function should be added to the `correct_image_illumination` docstring?